src/core/nodeOps.ts

```diff
@@ -33,8 +32,9 @@ export function invalidateInstance(instance: TresObject) {
 
 }
 
-export const nodeOps: RendererOptions<TresObject, TresObject | null> = {
-  createElement(tag, _isSVG, _anchor, props): TresObject | null {
+export const nodeOps: () => RendererOptions<TresObject, TresObject | null> = () => {
```

Comment:
(non-blocking)

Maybe use a verb now, since this is a function? Rename to `getNodeOps`?